src/app/modules/main/communities/tokens/controller.nim

```diff
@@ -76,6 +82,9 @@ proc computeAirdropCollectiblesFee*(self: Controller, collectiblesAndAmounts: se
 proc selfDestructCollectibles*(self: Controller, communityId: string, password: string, walletAndAmounts: seq[WalletAndAmount], contractUniqueKey: string) =
   self.communityTokensService.selfDestructCollectibles(communityId, password, walletAndAmounts, contractUniqueKey)
 
+proc burnCollectibles*(self: Controller, communityId: string, password: string, contractUniqueKey: string, amount: int) =
```

Comment:
Can this be `burnToken`, since the same params will be needed also for burning assets?

Reply:
Probably it will be when I add assets.